main.go

```diff
@@ -2,71 +2,132 @@ package main
 
 import (
 	"bytes"
+	_ "embed"
 	"flag"
 	"fmt"
 	"log"
 	"os"
 	"text/template"
 
 	"github.com/nicheinc/mock/iface"
+	"golang.org/x/tools/go/packages"
 	"golang.org/x/tools/imports"
 )
 
+//go:embed template.tmpl
+var tmpl string
```

Comment:
I renamed `template.go` to `template.tmpl` and embedded it in this `tmpl` variable, which makes the template file a little neater (no giant raw string literal).